web/handlers/__init__.py

```diff
@@ -103,6 +104,17 @@ async def get(self):
         output = template.render(Context=json.dumps({"List": apilist}))
         self.finish(output)
 
+    async def head(self):
+        root = self.biothings.config._primary
+        attrs = [getattr(root, attr) for attr in dir(root)]
+        confs = [attr for attr in attrs if isinstance(attr, types.ModuleType)]
+        apilist = [{"_id": conf.API_PREFIX, "status": "running"} for conf in confs]
+
+        templateEnv.globals["site"] = hostname_to_site(self.request.host)
+        template = templateEnv.get_template("index.html")
+        output = template.render(Context=json.dumps({"List": apilist}))
+        self.finish(output)
+
```

Comment:
since the block is the same as `get` method, shall we put it as a separate method and then call the same method in both `get` and `head`?

And for `head` it should not return any content, so should be `self.finish()`?